nl_server/config.py

```diff
@@ -99,13 +104,30 @@ class EmbeddingsConfig:
   models: Dict[str, ModelConfig]
 
 
+#
+# Get Dict of vertex ai model to its info
+#
+def _get_vertex_ai_model_map() -> Dict[str, any]:
+  if os.path.exists(_VERTEX_AI_MODEL_CONFIG_PATH):
+    with open(_VERTEX_AI_MODEL_CONFIG_PATH) as f:
+      return json.load(f) or {}
+  else:
+    autopush_env_values = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
```

Comment:
nit: Can do the following:

from pathlib import Path
current_file_path = Path(__file__)
autopush_env_values = current_file_path.parent.parent / 'deploy/helm_charts/envs/autopush.yaml'

Reply:
done